update_chroot

```diff
@@ -58,7 +66,7 @@ sudo touch "/etc/portage/make.conf.user"
 sudo_clobber "/etc/portage/make.conf" <<EOF
 # Created by update_chroot
 PORTDIR="${PORTAGE_STABLE_OVERLAY}"
-PORTDIR_OVERLAY="${CROSSDEV_OVERLAY} ${COREOS_OVERLAY}"
+PORTDIR_OVERLAY="${CROSSDEV_OVERLAY} ${COREOS_OVERLAY} ${EXTRA_PORTDIR_OVERLAY}"
```

Comment:
I didn't think about repos.conf ... that could have saved some time since the entire point is to enable this 'extra dir' for anyone looking to 'bolt on' to CoreOS, and backtracking through the build code, the PORTDIR_OVERLAY flag in make.conf seemed the place to fix.

I did think about the 'wipeout' problem and decided that anyone using these options would be smart enough to ensure they keep the command line flags in order and used the option everywhere necessary. But repos.conf does offer a much cleaner way.